modules/get-started/pages/whats-new.adoc

```diff
@@ -11,22 +11,33 @@ See also: xref:deploy:deployment-option/cloud/whats-new-cloud.adoc[]
 
 == Data transforms
 
-* Output to multiple topics: Data transforms can now output to multiple topics, for use cases that might require distributing different message types to multiple event streams, or for writing valid records to an output topic and invalid records to a dead-letter queue. See the xref:reference:data-transform-golang-sdk.adoc[Go] or xref:reference:data-transform-rust-sdk.adoc[Rust] SDK reference for data transforms for implementation details.
+* Output to multiple topics: Data transforms can now output to multiple topics.  For example, you can distribute different message types to multiple event streams, or you can write valid records to an output topic and invalid records to a dead-letter queue. See the xref:reference:data-transform-golang-sdk.adoc[Golang SDK] and xref:reference:data-transform-rust-sdk.adoc[Rust SDK] for data transforms.
 
-* xref:develop:data-transforms/run-transforms.adoc#view-data-transform-logs[Data transform logs] are now stored in an internally-managed topic and are accessible via `rpk` and the Redpanda Console.
+* xref:develop:data-transforms/run-transforms.adoc#view-data-transform-logs[Data transform logs] are now stored in an internally-managed topic and are accessible via `rpk` and Redpanda Console.
 
 == Topic-aware partition balancing 
 
 Redpanda now prioritizes xref:manage:cluster-maintenance/cluster-balancing.adoc[balancing a topic's partition replica count on each broker] while it's balancing the cluster's overall partition count. Because different topics in a cluster can have vastly different load profiles, this better distributes the workload evenly across brokers. This feature is enabled by default with the `partition_autobalancing_topic_aware` property. After upgrading to 24.1, new topics are distributed in a topic-aware fashion, and the next partition rebalance (triggered by adding new brokers or by calling the `/v1/partitions/rebalance` Admin API endpoint) evens out the partition counts on each broker.
 
+== Managed identities for Azure
+Redpanda now supports managed identities for Azure resources, similar to the existing support for IAM roles for self-hosted deployments on AWS and GCP. This provides a safer alternative to static credentials, like access keys. Managed identities allow Redpanda clusters deployed on Azure virtual machines and on Azure Kubernetes Service (AKS) to access Azure Blob Storage (ABS) and other services without storing credentials in the cluster configuration.
+
+== Topic recovery validation
+
+The properties xref:reference:cluster-properties.adoc#cloud_storage_recovery_topic_validation_mode[`cloud_storage_recovery_topic_validation_mode`] and xref:reference:cluster-properties.adoc#cloud_storage_recovery_topic_validation_depth[`cloud_storage_recovery_topic_validation_depth`] now add a consistency check that runs before a `redpanda.remote.recovery=true` topic is recovered from object storage.
```

Comment:
@andrwng  doesn't this  validation also occur with Whole Cluster Recovery? If so, we should call that out @micheleRP 

Basically both Whole Cluster Recovery and Topic Recovery perform this integrity check of the manifests in Tiered Storage.

Defer to @andrwng  on the exact wording.

Also, if it's on by default ( I hope it is), we might just say that recovery _does_ this by default and leave the conf property details to the feature docs

Reply:
I think this is only covered in the property reference (i.e., no feature docs): Is that correct @Deflaimun?

Reply:
Right, the check happens for cluster restore and topic recovery alike.

IIUC the default behavior is to just check that the partition manifests are downloadable, and the `cloud_storage_recovery_topic_validation_mode` property must be set to `check_manifest_and_segment_metadata` to get deeper validation of segments. @andijcr should be able to speak definitively.

Reply:
Thanks @andrwng! @andijcr please review the updated section [Validation before topic recovery or cluster restore](https://deploy-preview-450--redpanda-docs-preview.netlify.app/24.1/get-started/whats-new/#validation-before-topic-recovery-or-cluster-restore)

Reply:
yes @andrwng is correct

Reply:
Thanks @andijcr! I've updated the section: 
Before topic recovery or cluster restore, the properties [cloud_storage_recovery_topic_validation_mode](https://deploy-preview-450--redpanda-docs-preview.netlify.app/24.1/reference/cluster-properties/#cloud_storage_recovery_topic_validation_mode) and [cloud_storage_recovery_topic_validation_depth](https://deploy-preview-450--redpanda-docs-preview.netlify.app/24.1/reference/cluster-properties/#cloud_storage_recovery_topic_validation_depth) now check that partition manifests exist and confirm that there are no connection issues to object storage.